package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "I used what we’ve learned and take a chance to continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage! At the end, we use use JavaScript alone to generate the entire contents of the website!",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
## Add missing scripts 
- [ ] Ensure we have all the necessary run common commands. We only have `build`, here `package.json` inside \`scripts'. take a look on here 👇 as a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
```suggestion
   "scripts": {
       "dev": "webpack --mode development",
       "build": "webpack --mode production",
       "start": "webpack-dev-server --mode development --open" // optional
     },
```